File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldDocument.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnDocFileExt;
import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioFieldDocument extends StudioFieldEditable
{
  @Nullable
  public EnumDefnDocFileExt[] fileTypeSet;

  @Nullable
  public Long maxSize;

  @Nullable
  public MetaIdField maxSizeFieldId;

  @Nullable
  public MetaIdVar maxSizeVarId;

  @Nullable
  public Boolean showSize;

  @Nullable
  public MetaIdField showSizeFieldId;

  @Nullable
  public MetaIdVar showSizeVarId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldDocument.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnDocFileExt
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioFieldEditable

interface StudioFieldDocument : StudioFieldEditable
{
  val fileTypeSet: Array<EnumDefnDocFileExt>?
  val maxSize: Long?
  val maxSizeFieldId: MetaIdField?
  val maxSizeVarId: MetaIdVar?
  val showSize: Boolean?
  val showSizeFieldId: MetaIdField?
  val showSizeVarId: MetaIdVar?
}